.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,45 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+1. - [ ] Update target branch to new version.
+   - For 2.x:
+     - [ ] If releasing 2.x, run auto-update (refer to docs).
+   - For 3.x:
+     - [ ] if maestro auto-PR is active, then verify SHA1s in version.Details with manifest versions in VSU share dir (refer docs), make sure that we match the versions
+     - [ ] in-case auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
```

Comment:
(Nit) Sounds like a preventative measure with current wording, but should only be done if condition is met

```suggestion
     - [ ] if auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
```